src/styles/main.scss

```diff
@@ -30,6 +30,11 @@ a {
     text-decoration: none;
 }
 
+.text-content {
```

Comment:
@zroehr: Can you confirm if there's a max width for text on the site, if it would apply to this form, and what value it should be?